rdir/rdir.c

```diff
@@ -991,6 +991,38 @@ _request_to_key(struct json_object *jbody, GString **pkey)
 	return NULL;
 }
 
+
+// RDIR{{
+// DELETE /v1/rdir/delete?vol=<volume ip>%3A<volume port>
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// Delete the target volume.
```

Comment:
Unfortunately, this does not delete any "volume". It unreferences a chunk from the volume. To be clearer, I suggest putting the JSON body under the headers (for all request examples).